src/components/organisms/deployments/sessions/viewer.tsx

```diff
@@ -340,17 +346,36 @@ export const SessionViewer = () => {
 								classIcon="fill-none group-hover:fill-none group-hover:stroke-green-800 stroke-white size-5 mb-0.5"
 								closeIcon={CircleMinusIcon}
 								openIcon={CirclePlusIcon}
-								title="Inputs"
+								title={t("inputs")}
 							>
 								<JsonView
-									className="scrollbar max-h-72 overflow-auto"
+									className="scrollbar max-h-72 overflow-auto rounded-md border border-gray-1000 !bg-transparent p-2"
 									style={githubDarkTheme}
 									value={sessionInfo.inputs}
 								/>
 							</Accordion>
 						</div>
 					) : null}
 				</div>
+				<div className="flex-1">
+					{sessionInfo.outputs ? (
+						<div className="mt-3 max-w-[80%] pb-3.5">
+							<Accordion
+								classChildren="border-none pt-3 pb-0"
+								classIcon="fill-none group-hover:fill-none group-hover:stroke-green-800 stroke-white size-5 mb-0.5"
+								closeIcon={CircleMinusIcon}
+								openIcon={CirclePlusIcon}
+								title={t("outputs")}
+							>
+								<JsonView
+									className="scrollbar max-h-72 overflow-auto rounded-md border border-gray-1000 !bg-transparent p-2"
+									style={githubDarkTheme}
+									value={sessionInfo.outputs}
```

Comment:
Outputs should display only:
`CompletedLogRecord?.state?.completed?.returnValue`